### PR TITLE
doppler: add page

### PR DESCRIPTION
**Pull request by thewolmer**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known): idk 😐**

### CODE REVIEW

File: pages/common/doppler-projects.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+> Manage Doppler Projects.
+> More information: <https://docs.doppler.com/docs/cli>.
+
+- Get all projects:
+
+`doppler projects`
+
+- Get info for a project:
+
+`doppler projects get {{name/project_id}}`
```
Comment (spageektti) [inline pages/common/doppler-projects.md:12]: Does this require both name and project_id, or only one of them?

```suggestion
`doppler projects get {{name|project_id}}`
```

File: pages/common/doppler-secrets.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Upload a secrets file:
+
+`doppler secrets upload {{./path/to/file.env}}`
+
+- Delete value(s) of one or more secrets:
+
+`doppler secrets delete {{secrets}}`
+
+- Download secrets as .env:
```
Comment (spageektti) [inline pages/common/doppler-secrets.md:22]:
```suggestion
- Download secrets as `.env`:
```

**Review by spageektti — COMMENTED** [review]
Hi. Thanks for your contribution,

File: pages/common/doppler-projects.md (hunk 1 above)
Comment (thewolmer) [reply pages/common/doppler-projects.md]: It requires only one of them. Your suggestion `{{name|project_id}}` is the correct one i believe.
Thanks 🫡

File: pages/common/doppler-projects.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`doppler projects get {{name|project_id}}`
+
+- Create a project:
+
+`doppler projects create {{name}}`
+
+- Update a project's name and description:
+
+`doppler projects update {{name/project_id}} --name "{{new_name}}" --description "{{new_description}}"`
```
Comment (spageektti) [inline pages/common/doppler-projects.md:20]:
```suggestion
`doppler projects update {{name|project_id}} --name "{{new_name}}" --description "{{new_description}}"`
```

**Review by spageektti — APPROVED** [review]
LGTM.

**Pull request merged**